### DISCUSSION TITLE
connect pretix with 3rd ERP system

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3156] [Q&A]
Hello pretix team!

I would like to connect a pretix community implementation with an ERP system for the following use cases.

1. A visitor of Pretix buys a ticket by placing an order.
If this user is an existing one in the ERP, then the order details will be sent to the ERP and will update the existing customer.
If this is a new user, then a new customer will be created in the ERP and then the order details will update the ERP in the corresponding fields.

2. Some users of the ERP system will have some vouchers issued by Pretix. These vouchers can be used for specific number of times and for specific time period, after which they will be invalid.

I would like to know what are the table/tables of the database that each time the above incidents occur, these tables are updated.

Thank you in advance

**Comment by raphaelm** [comment]
I'd recommend using the REST API instead of the database, as it is documented and backwards-compatible between pretix versions.

**Comment by alemao8** [comment]
Hello Raphael and thank you for the response.
Could you provide me a sample payload for the above if it's not much trouble ?
Thanks again for your support.

**Reply by raphaelm** [reply]
No, sorry, just what's in the docs.
